code/baselines/wsdbaselines_track1_table.py

```diff
@@ -0,0 +1,30 @@
+import argparse
+
+import pandas as pd
+import numpy as np
+import sys
+from pathlib import Path
+
+
+def main():
+    dfs = []
+    for p in Path('./').glob('*.scores.tsv'):
+        mdf = pd.read_csv(p, sep=': +', header=None, engine='python')
+        mdf.rename(columns={0: 'metric', 1: 'value'}, inplace=True)
+        ff = p.name.split('.')
+        mdf['method'] = ff[0]
+        mdf['lang'] = ff[1]
+        mdf['part'] = ff[2]
+        dfs.append(mdf)
+
+    mdf = pd.concat(dfs, ignore_index=True)
+    results = mdf.groupby(['method', 'lang', 'part', 'metric']).value.agg(lambda r:
+                                                                          f'{np.mean(r):.3f}+-{np.std(r):.3f} ({len(r)})').reset_index()
+
+    results = results.pivot_table(index=['method', 'part'], columns=['metric', 'lang'], values='value',
+                                  aggfunc=lambda r: ' '.join(r))
+    results.columns = [f'{c2}-{c1}' for c1, c2 in results.columns]
+    results.to_csv('wsdbaselines.scores.tsv', sep='\t', index=True)
```

Comment:
As it is now, if this script is run more than once (e.g. if a user cloned your repo with already pushed wsdbaselines.scores.tsv), this result file goes into the loop of all '.scores.tsv' in this path and  its format gets incorrect. If you will move all '*.score.tsv' named after method, language and split into a separate folder as recommended above, the problem will be solved.

Reply:
done

Reply:
Thank you, the problem is gone. I got some metrics slightly different, e.g. 0.044 instead of 0.042 . Probably this is because of numerical instability and not a big deal

Reply:
This is expected for to randomness. The empirical mean should be within 2-3 std*sqrt(#trials) from the real mean, so the difference within .004-..006 between runs is normal.

Reply:
std is reported in the table, one can just check based on that

Reply:
Stage this happens despite I've set the random seen for sampling, probably due to the difference in versions of the libraries. However, I wouldn't elaborate on exactly reproduction of the results for these random baselines unless the difference is statistically significant (outside the confidence intervals)